integration-tests/on-device-diagnose/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import TestSuite from './TestSuite'
+import { BleManager } from 'react-native-ble-plx'
+
+export default class Diagnose extends React.Component {
+  render () {
+    return (
+      <TestSuite testCases={[
+        ["Initialize BleManager", () => {
+          if (!BleManager) {
+            throw new Error('BleManager is null!');
+          }
```

Comment:
Put it inside promise with reject result?

Reply:
It will be fixed by fixing #413